cmd/uplink/cmd/cp.go

```diff
@@ -191,7 +196,7 @@ func copyMain(cmd *cobra.Command, args []string) (err error) {
 	}
 
 	// if uploading
-	if u0.Scheme == "" {
+	if filepath.IsAbs(args[0]) || u0.Scheme == "" {
```

Comment:
Rather than checking here, fix ParseURL to handle Windows filepaths properly.

Reply:
Yeah, might be a better approach.